tests/admin/test_actions.py

```diff
@@ -47,6 +48,60 @@ class Meta(object):
 
 
 @mark.django_db
+class TestClearCatalogAction(unittest.TestCase):
+    """
+    Tests for clear_catalog_id action.
+    """
+    def setUp(self):
+        """
+        Provide the necessary moving parts for the tests.
+        """
+        self.action = get_clear_catalog_id_action()
+        self.model_admin_mock = mock.Mock()
+        self.model_admin_mock.model._meta.fields = [
+            self._make_field("code"), self._make_field("name"), self._make_field("description"),
+        ]
+        super(TestClearCatalogAction, self).setUp()
+
+    def test_existing_value_removed(self):
+        """
+        Test that catalog IDs are cleared appropriately.
+        """
+        for _ in range(5):
+            EnterpriseCustomerFactory()
+        query_set = EnterpriseCustomer.objects.all()  # pylint: disable=no-member
+        for customer in query_set:
+            self.assertNotEqual(customer.catalog, None)
```

Comment:
@haikuginger pytest assertions please - I'm trying to standardize on using them in this repo.

If this file uses unittest assertions - it would be very nice if you could replace them with pytest assertions, but that's not a requirement to land this PR.